### DISCUSSION TITLE
Dial borders

### DISCUSSION THREAD

**Discussion by dgcasey0325** [fastaddons/GroupSpeedDial#251] [General]
This should be an easy one to fix. Yesterday something happened with my Dials and I lost over half the thumbnails. I restored using the import of a backup and everything worked except for one thing. Now I have faded white borders around each dial and can't find the option to turn the borders off. Can someone tell be where that option is, please? Thank you.

![Image3](https://github.com/fastaddons/GroupSpeedDial/assets/38332950/28cc149a-d88b-4659-8e1f-17bcc4ae38bb)

**Comment by Juraj-Masiar** [comment]
Hello,
Yes, I've broke it in 22.6. Fix is already being deployed and waiting for Chrome Store review, more info here:
https://github.com/fastaddons/GroupSpeedDial/issues/250

I'm sorry for the complications!

**Comment by dgcasey0325** [comment]
Thanks. I'll keep an eye for it.

**Comment by Juraj-Masiar** [comment]
22.7 has been released
